open_src/github.com/VictoriaMetrics/VictoriaMetrics/lib/fs/reader_mmap_windows.go

```diff
@@ -0,0 +1,6 @@
+package fs
+
+// SetDisableMmap set disableMmap
+func SetDisableMmap(off bool) {
+	disableMmap = off
+}
```

Comment:
No need to set mmap for windows.

```suggestion
func SetDisableMmap(off bool) {}
```